build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    kotlin("jvm") version "1.4.0"
```

Comment:
the kotlin plugin applies the java plugin, so no need to do that separately. (Java and Kotlin source will both be compiled)